code/datums/balloon_alerts/balloon_alerts.dm

```diff
@@ -25,8 +37,7 @@
 	for(var/mob/hearer in hearers)
 		if(is_blind(hearer))
 			continue
-
-		balloon_alert(hearer, (hearer == src && self_message) || message, text_color)
+		balloon_alert(hearer, (hearer == src && self_message) || message, text_color, delay)
```

Comment:
if we have multiple viewers, only the first viewer will be able to see the alert, currently, no?